raiden/accounts.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import json
+import getpass
+
+from pyethapp.accounts import Account
+
+
+def find_datadir():
+    home = os.path.expanduser("~")
+    datadir = None
+    if sys.platform.startswith('linux'):
+        datadir = os.path.join(home, '.ethereum')
+    elif sys.platform == 'darwin':
+        datadir = os.path.join(home, 'Library', 'Ethereum')
+    else:
+        raise RuntimeError('Unsupported Operating System')
+
+    if os.path.isdir(datadir) is False:
```

Comment:
I understand your point here but at the moment raiden does not do any account managing on its own whatsoever. It's supposed to be read-only, so we would never create the directory ourselves.

If we want to this can easily be changed/modified in a subsequent PR.